php/php.editor/src/org/netbeans/modules/php/editor/model/impl/ModelVisitor.java

```diff
@@ -1117,7 +1117,7 @@ public void visit(CatchClause node) {
         Variable variable = node.getVariable();
         Scope scope = modelBuilder.getCurrentScope();
         List<Expression> classNames = node.getClassNames();
-        if (scope instanceof VariableNameFactory) {
+        if (variable != null && scope instanceof VariableNameFactory) {
```

Comment:
Not needed, `null` is not `instanceof`, ever.

Reply:
Probably, in this case, needed because variables(`variable` and `scope`) are different?

Reply:
Sorry, those are 2 different variables... :)

Reply:
No problem :)